consensus/src/quorum_store/quorum_store_db.rs

```diff
@@ -67,7 +36,22 @@ impl QuorumStoreDB {
         Self { db }
     }
 
-    pub(crate) fn delete(&self, digests: Vec<HashValue>) -> Result<(), DbError> {
+    pub(crate) fn get_batch_id(&self, epoch: u64) -> Result<Option<BatchId>, DbError> {
+        Ok(self.db.get::<BatchIdSchema>(&epoch)?)
+    }
+
+    pub(crate) fn delete_batch_id(&self, epoch: u64) -> Result<(), DbError> {
```

Comment:
this could be private based on our discussion, if there is one place that's supposed to get the max, it can also clean smaller epochs.